oak_functions/loader/fuzz/fuzz_targets/wasm_invoke.rs

```diff
@@ -106,11 +119,16 @@ impl From<&ArbitraryInstruction> for crate::proto::Instruction {
                     response: response.clone(),
                 }),
             ),
-            ArbitraryInstruction::StorageGetItem { key } => {
-                Some(InstructionVariant::StorageGetItem(
+            ArbitraryInstruction::StorageGetItem { key } => match key {
+                LookupKey::Key => Some(InstructionVariant::StorageGetItem(
+                    crate::proto::StorageGetItem {
+                        key: FIXED_KEY.to_vec(),
```

Comment:
It looks like the only thing that needs to change between the two match branches is the actual key, but there is no need to repeat all the rest of the expression, right?

Reply:
Fixed.